modules/tchap-translations/README.md

```diff
@@ -0,0 +1,38 @@
+# Tchap Translations - a long story
+
+## What are all these translation files ?
+
+-   `tchap_translations_web.json` and `tchap_translations_react-sdk.json` : translations for strings that Tchap code adds to the web and react-sdk repos. Strings added in patch code should be translated in the react-sdk file. The rest in the web file.
```

Comment:
I would merge those two files in another PR to simplify the maintenance

Reply:
done